tests/unit/Diff/EntityDiffOldTest.php

```diff
@@ -127,9 +127,14 @@ public static function generateApplyData( $entityType ) {
 	}
 
 	/**
+	 * @return array[]
+	 */
+	public abstract function provideApplyData();
+
+	/**
 	 * @dataProvider provideApplyData
 	 */
-	public function testApply( Entity $a, Entity $b ) {
+	public function testApply( Item $a, Item $b ) {
```

Comment:
Obviously a design issue, but since Item is not introduced in this patch (see the use section), this is a very nice, minimal migration path.